manual/src/html_processing/src/common.ml.in

```diff
@@ -27,21 +27,23 @@ let ( // ) = Filename.concat
 
 let process_dir = Filename.current_dir_name
 
+let ocaml_version = "5.3.0"
```

Comment:
The manual is shared between patch version, so `5.3` is probably better. And this version number is available as a configure time macro as `"@OCAML_VERSION_SHORT@"`

Reply:
We want to make the manuals available for all OCaml releases, which includes minor versions as well. Hence, the x.y.z naming convention has been used for consistency.

Reply:
Except that bugfix versions share the same features, manual and API that their main version. I am not convinced that it is useful to differentiate between them for the manual.

Reply:
Okay. I will build using `"@OCAML_VERSION_SHORT@"` and shall update the PR. I now see the version handling at ocaml.org for version numbering at https://github.com/ocaml/ocaml.org/blob/86acab21cac4f887d2ce2b0d963813a78e3fa81d/src/global/url.ml#L42-L51. Thank you!